docs/tutorial/tutorial.md

```diff
@@ -132,7 +132,7 @@ After: You should see a number in each square in the rendered output.
 
 ### An Interactive Component
 
-Let's make the Square component fill in an "X" when you click it. Try changing the button tag returned in the `render()` function of the `Square` like this:
```

Comment:
There was inconsistent use of backticks in component names. Removing them reduces some noise.